interpreter/golabels/tls_amd64.go

```diff
@@ -44,35 +51,57 @@ func extractTLSGOffset(f *pfelf.File) (int32, error) {
 	if err != nil {
 		return 0, err
 	}
+	return inspectCode(sym, f.VirtualMemory, code)
+}
 
+func inspectCode(sym *libpf.Symbol, extract readMemory, code []byte) (int32, error) {
 	offset := e.NewImmediateCapture("offset")
 	it := amd.NewInterpreterWithCode(code)
+	pc := 0
+	// Register -> loaded TLS offset from RIP-relative memory
+	ripRelLoads := make(map[x86asm.Reg]int64)
+
 	for {
 		op, err := it.Step()
 		if err != nil {
 			break
 		}
+		pc += op.Len
```

Comment:
The `it asm.amd.Interpreter` object tracks already `pc`. Could you instead add `PC` accessor to it to read it, and use it? I think adding it there would be useful in future.